lib/scout_apm/plugs/controller_timer.ex

```diff
@@ -54,20 +54,22 @@ defmodule ScoutApm.Plugs.ControllerTimer do
 
   def maybe_mark_error(conn), do: conn
 
-  # Takes a connection, extracts the phoenix controller & action, then manipulates & cleans it up.
-  # Returns a string like "PageController#index"
+  @doc """
+  Takes a connection, extracts the phoenix controller & action, then manipulates
+  & cleans it up.
+
+  Returns a string like "PageController#index"
+  """
+  @spec action_name(Plug.Conn.t()) :: String.t()
   def action_name(conn) do
-    controller_name = conn.private[:phoenix_controller]
     action_name = conn.private[:phoenix_action]
 
-    # a string like "Elixir.TestappPhoenix.PageController#index"
-    "#{controller_name}##{action_name}"
-    # Split into a list
-    |> String.split(".")
-    # drop "Elixir.TestappPhoenix", leaving just ["PageController#index"]
-    |> Enum.drop(2)
-    # Probably just "joining" a 1 elem array, but recombine this way anyway in case of periods
+    conn.private[:phoenix_controller]
+    |> Module.split()
```

Comment:
Good call on  'Module.split`, just double checked its history because we have to support back to rather old versions of Elixir, and it's been there since before 1.2 era, which is further back than we need. Not sure how we missed that function when writing this code originally.